oaknational/Oak-Web-Application
# Update Curriculum Visualizer Tab

## Description

- Added UI elements to visualize curriculum units in curriculum `Unit Sequence` tab
- Integrated Curriculum 2023 API for unit data
- Updated subject phase picker to pre-select for current view
- Deleted legacy subjects from subject phase picker

## How to test

1. Go to https://deploy-preview-1798--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum/maths-secondary/units
2. You should see a grid of the units in the maths secondary curriculum sequence
3. For years 
4. You should see "Maths" and "Secondary" selected in the subject phase picker
5. In the subject phase picker, click on "Geography" then "Primary"
6. You should see a grid of the units in the geography primary curriculum sequence
7. Go to https://deploy-preview-1798--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum/science-secondary/units
8. For year 10 and year 11, you should see filters for child subjects and tiers
9. Click "Physics" subject and "Higher" tier
10. You should see the units updated accordingly
11. Go to: https://deploy-preview-1798--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum/english-primary/units
12. You should see domains for all years
13. Click "Grammar" for year 1
14. You should see the units updated accordingly

## Checklist
